_playwright-tests/tests/API/Repositories.spec.ts

```diff
@@ -207,4 +212,64 @@ test.describe('Repositories', () => {
       expect(json.url?.httpCode).toEqual(0);
     });
   });
+
+  test('Verify RPM search', async ({ client }) => {
+    let rpmSearch: ApiSearchRpmResponse[];
+    await test.step('Search small red hat repo', async () => {
+      rpmSearch = await new RpmsApi(client).searchRpm({
+        apiContentUnitSearchRequest: {
+          search: 'ansible',
+          urls: [SmallRedHatRepoURL],
+        },
+      });
+      const names = rpmSearch.map((v) => v.packageName);
+      expect(names).toContain('ansible');
+    });
+  });
+
+  test('Add popular repository', async ({ client }) => {
+    let popularRepos: ApiPopularRepositoriesCollectionResponse;
+    await test.step('list popular repositories', async () => {
```

Comment:
tiny nitpick, feel free to ignore 😅 _most of our tests and steps start with capital letters which makes it a tiny bit nicer in the report, keeping it uniform would be nice ✨_